styles/05-objects/avatar.css

```diff
@@ -0,0 +1,28 @@
+.avatar{
+  aspect-ratio: 1/1;
+  position: relative;
+}
+.avatar-status{
+  aspect-ratio: 1/1;
+  border: 3px solid;
+  position: relative;
+}
+.avatar-xs{
+  width: 2rem;
+  height: 2rem;
+}
+.avatar-sm{
+  width: 4rem;
+  height: 4rem;
+}
+.avatar-lg{
+  width: 8rem;
+  height: 8rem;
+}
+.avatar-md{
+  width: 6rem;
+  height:6rem;
+}
+.round{
```

Comment:
This can be `.avatar-round` if it makes sense. Or if this is generic, can be moved to utilities.css.

Reply:
I'll think about it

Reply:
Its generic. i'll move it to utils